src/SnD.Sdk.Storage.Minio/Providers/MinioStorageProvider.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Minio;
+using SnD.Sdk.Storage.Minio.Base;
+using Snd.Sdk.Storage.Minio.Providers.Configurations;
+
+namespace Snd.Sdk.Storage.Minio.Providers;
+
+public static class MinioStorageProvider
+{
+
+    public static IServiceCollection AddMinioStorage(this IServiceCollection services, IConfiguration appConfiguration)
```

Comment:
I would suggest to do it the same way as it is in [Aws](https://github.com/SneaksAndData/esd-services-sdk/blob/4f45d5a7c2f3d68e32b71492b004b7fb842acca2/src/SnD.Sdk/Storage/Providers/AmazonStorageServiceProvider.cs#L23) and [Azure ](https://github.com/SneaksAndData/esd-services-sdk/blob/4f45d5a7c2f3d68e32b71492b004b7fb842acca2/src/SnD.Sdk/Storage/Providers/AzureStorageServiceProvider.cs#L52): accept the `MinioConfiguration` instance, not the IConfiguration.